src/components/Form.js

```diff
@@ -82,6 +82,11 @@ const propTypes = {
     /** Information about the network */
     network: networkPropTypes.isRequired,
 
+    /** Style for the button */
+    submitButtonStyle: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.object), PropTypes.object]),
```

Comment:
`StyleUtils.parseStyleAsArray` will also be useful here, I think!

Reply:
👍🏻